config/API/views.py

```diff
@@ -16,3 +16,15 @@ def get(self, request):
             'description': 'An API for BetterCalendar project. Currently in development - this is the only view.',
         }
         return Response(data, status=status.HTTP_200_OK)
+    
+class APIDBTestView(APIView):
+    def post(self, request):
+        user = UserData()
+        user.e_mail = "test@test.com"
+        user.user_name = "testuser"
+        user.save()
+        return Response(status=status.HTTP_200_OK)
+
+    def get(self, request):
+        data = UserData.objects.all().values()
```

Comment:
Tu bym od razu to zmienił, żeby przypadkiem nie zostało - niech zwraca usera takiego jak wyżej.

Reply:
Te dwie metody będą i tak ostatecznie będą do usunięcia. Używałem ich do testowania zapisu i odczytu z bazy (post dodaje usera ze stałymi danymi, a get zwraca wszystkie elementy z kolekcji userdata), a zostawiłem bo myślałem, że się komuś przyda do sprawdzenia połączenia z bazą.

Masz na myśli tworzenie nowego usera w metodzie get, tak jak w metodzie post i zwracanie go?
```
def get(self, request):
        user = UserData()
        user.e_mail = "test@test.com"
        user.user_name = "testuser"
        user.save()
        return Response(user, status=status.HTTP_200_OK)
```

Bo w takim przypadku to trochę mija się z pierwotnym przeznaczeniem tej metody. Ewentualnie mogę je już teraz usunąć.

Reply:
Chodziło mi o wyeliminowanie kodu `data = UserData.objects.all().values()`.  
Zrób, żeby get zwracało tylko tego testowego użytkownika czy coś.  
Bo na potrzeby developmentu imo taki endpoint może zostać. Tylko niech nie będzie furtką do wyciągnięcia wszystkich userów :P